source/administration/config-settings.rst

```diff
@@ -3863,6 +3863,20 @@ Enable API Team Deletion
 | This feature's ``config.json`` setting is ``"EnableAPITeamDeletion": false`` with options ``true`` and ``false``.                                                    |
 +----------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 
+Enable OpenTracing
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**True**: A Jaeger client is instantiated and each HTTP request wraps App and Store in OpenTracing layers.
+Context is added to App and Store and is passed down the layer chain to create OpenTracing 'spans'.
```

Comment:
I'd phrase it a little bit differently:
"A [Jaeger](https://www.jaegertracing.io/) client is instantiated and is used to trace each HTTP request as it goes through App and Store layers."